internal/host/logscmd.go

```diff
@@ -44,3 +56,29 @@ func (i *logsCmd) GetSteps(ctx context.Context, host *models.Host) ([]*models.St
 
 	return []*models.Step{step}, nil
 }
+
+func (i *logsCmd) getMastersIPsInHostCluster(ctx context.Context, host *models.Host) ([]string, error) {
+
+	var cluster models.Cluster
+	if err := i.db.First(&cluster, "cluster_id = ?", host.ClusterID).Error; err != nil {
+		i.log.WithError(err).Errorf("failed to get cluster for host %s", host.ID)
+		return nil, err
+	}
+
+	var hosts []*models.Host
+	if err := i.db.Find(&hosts, "cluster_id = ?", host.ClusterID).Error; err != nil {
+		i.log.WithError(err).Errorf("failed to get list of hosts for cluster %s", host.ClusterID)
+		return nil, err
+	}
+
+	var ips []string
+	for _, h := range hosts {
+		ip, err := network.GetMachineCIDRIP(h, &common.Cluster{Cluster: cluster})
```

Comment:
and if u have list of ips?

Reply:
I return the first one that matches, not a list of Its

Reply:
just to be sure i understand. U return one that matches cluster cidr?

Reply:
need to think how it will work with None platform. There will be no cidr in cluster. When u push this, open ticket for it please

Reply:
Please open the ticket under MGMT-2736

Reply:
> just to be sure i understand. U return one that matches cluster cidr?
Yes.

> Please open the ticket under MGMT-2736
Done.